ui/packages/app/src/Blocks.svelte

```diff
@@ -149,6 +149,14 @@
 
 	async function handle_mount() {
 		await tick();
+
```

Comment:
why?

Reply:
Because we want to open them all in a new tab (including any links that are generated via markdown, which is why we need to do it this way), this was previously being done with the `base` element but that _also_ affects form actions which we don't want because of the login form in iframes.